src/test/java/bio/terra/integration/BigQueryFixtures.java

```diff
@@ -62,76 +55,6 @@ public static boolean datasetExists(BigQuery bigQuery, String projectId, String
     }
   }
 
-  public static TableResult query(String sql, BigQuery bigQuery) {
```

Comment:
It brings me a LOT of joy to see these fixtures removed!

Reply:
Me too! Although, I haven't seen the test timing improvements I was hoping for, but I still think this is a better approach.